EMMO/README.md

```diff
@@ -2,47 +2,71 @@
 This [W3ID](https://w3id.org) provides a persistent URI namespace for the [Elementary Multiperspective Material Ontology (EMMO)](https://github.com/emmo-repo/EMMO/) and its domain ontologies.
 
 ## Redirection Rules
-This section contains a general summary of the logic behind the redirection rules.  
+This section contains a general summary of the logic behind the redirection rules.
 
 1. `https://w3id.org/emmo`
-   - If the user is accessing this from a browser: redirect to github pages documentation of this IRI. Otherwise, redirect to the `{DOMAIN}.ttl` file in the root of the master branch  
+   - If the user is accessing this from a browser: redirect to documentation of corresponding IRI (`https://w3id.org/emmo#{NAME}`).
+     Ex: http://industryportal.enit.fr/ontologies/EMMO?p=classes&conceptid=http%3A%2F%2Femmo.info%2Femmo%23EMMO_eb3a768e_d53e_4be9_a23b_0714833c36de
 
-2. `https://w3id.org/emmo#{NAME}`
-   - If the user is accessing this from a browser: redirect to github pages documentation of this IRI. Otherwise, redirect to the `{DOMAIN}.ttl` file in the root of the master branch  
+   - Otherwise, redirect to squashed ontology on GitHub Pages:
+[https://raw.githubusercontent.com/emmo-repo/emmo-repo.github.io/master/latest/emmo.ttl](https://raw.githubusercontent.com/emmo-repo/emmo-repo.github.io/master/versions/1.0.0-beta5/emmo.ttl)
 
-3. `https://w3id.org/emmo/`
-   - `{DOMAIN}.ttl` file in the root of the master branch  
+2. `https://w3id.org/emmo/`
+   - `emmo.ttl` file in the root of the master branch: https://raw.githubusercontent.com/emmo-repo/EMMO/master/emmo.ttl
 
-4. `https://w3id.org/emmo/{VERSION}`
-   - Given version of EMMO  
+3. `https://w3id.org/emmo/{VERSION}`
+   - If the user is accessing this from a browser: redirect to documentation of given version corresponding IRI (`https://w3id.org/emmo/{VERSION}#{NAME}`).
```

Comment:
The server strip off the hash and everything that follows it. Hence, the `#{NAME}`-part is not part of the redirection. But it may be used by the client to e.g. show a specific position of a web page. 

Please feel free to suggest improvements of the notation.